assets/src/editor/events/dates-and-times/editor-date/filter-bar/with-date-entities-list-filter-bar.js

```diff
@@ -45,7 +45,7 @@ export const getFilteredDateEntitiesList = ( dateEntities, showDates, datesSorte
  * @param {Object} EntityList
  * @return {Object} EntityList with added DateListFilterBar
  */
-export default createHigherOrderComponent(
+const withDateEntitiesListFilterBar = createHigherOrderComponent(
```

Comment:
curious why this change?  It was unnecessary?

Reply:
you're right... but that's kinda been the convention in every other file so I just changed it to match everything else.